lld/COFF/Writer.cpp

```diff
@@ -1566,6 +1567,30 @@ void Writer::createSymbolAndStringTable() {
   fileSize = alignTo(fileOff, ctx.config.fileAlign);
 }
 
+void Writer::mergeSection(const std::map<StringRef, StringRef>::value_type &p) {
+  StringRef toName = p.second;
```

Comment:
I assume this factorized function is just a direct move of the code?

Reply:
yes, with indents reduced and `continue`s turned into `returns`.